src/i18n/i18n.csv

```diff
@@ -102,6 +102,7 @@ Share widget,fav,Favourite,Lien préféré,Gunsteling,Pijet,المفضل,հով
 Form validation,frm-nosubmit,The form could not be submitted because ,Le formulaire n\'a pu être soumis car ,"Die vorm kon nie voorgel꡷ord nie, want ",Formulari nuk mund të paraqitet për shkak ,لا يمكن أن تقدم على شكل ل ,"Ձեւը չի կարող ներկայացվել, քանի որ ","Форма не могат да се подават, тъй като ",本表格不能傳送因為,表格无法提交，因为,"Formulář nelze předloženy, protože ",Het formulier kon niet worden verwerkt omdat ,"Ankeeti ei saa esitada, kuna","Das Formular konnte nicht vorgelegt werden, da ","Το έντυπο θα μπορούσε να να δεν που υποβάλλονται, επειδή ",फार्म प्रस्तुत नहीं किया जा सका क्योंकि ,Az adatokat nem lehet elküldeni mert ,Eyðublaðið var ekki hægt að skila því ,Il modulo non poteva essere presentata in quanto ,Non è stato possibile inviare il modulo in quanto ,ᑖᓐᓇ ᑎᑎᖅᑲᖅ ᐅᑎᖅᑎᒐᒃᓴᐅᙱᑦᑐᖅ ᐅᓇ ᐱᔾᔪᑕᐅᓪᓗᓂ,フォームを送信できませんでした。原因：,양식을 제출 할 수 없습니다 때문에 ,"Formu nav iespējams iesniegt, jo ","Formos pateikti negalima, nes ","Nie można było wysłać formularza, ponieważ ",O formulário não pode ser submetido porque ,O formulário não pode ser submetido porque ,"Формы не могут быть представлены, потому что ","Formulár nemožno predložené, pretože ",El formulario no pudo ser enviado debido a ,รูปแบบไม่สามารถส่งเพราะ ,"Form gönderilemiyor, çünkü ","Форма не подається через те, що",Dạng không thể được nộp bởi vì 
 ,errs-fnd, errors were found., erreurs ont été trouvées., foute is gevind nie., gabime u gjetën., تم العثور على أخطاء., սխալներ չեն հայտնաբերվել., бяха открити грешки.,內有錯誤。,找到错误。, chyby byly nalezeny., er zijn fouten opgetreden.,on leitud vigu., Fehler gefunden wurden., τα σφάλματα βρέθηκαν., त्रुटियों पाया गया., hibák léptek fel., villur fundust., error ditemukan., errori sono stati trovati.,ᑕᒻᒪᖅᓯᒪᔪᓂᑦ ᖃᐅᔨᔪᖅ,複数のエラーが見つかりました。, 오류가 발견되었습니다., tika atrastas kļūdas., rastos klaidos., znaleziono błędy, erros encontrados., foram encontrados erros., были обнаружены ошибки., chyby boli nájdené., errores encontrados., พบข้อผิดพลาด., hatalar bulundu.,знайдено помилки., lỗi đã được tìm thấy.
 ,err-fnd, error was found., erreur a été trouvée., fout gevind word., gabim u gjet., تم العثور على خطأ., սխալ է գտնվել., грешка е намерен.,出現錯誤。,找到错误。, chyba byla nalezena., er is een fout opgetreden.,on leitud viga., Fehler festgestellt wurde., σφάλμα ήταν που βρέθηκαν., त्रुटि पाया गया था., hiba lépett fel., villa fannst., error ditemukan., errore è stato trovato.,ᑕᒻᒪᖅᓯᒪᔪᒥᑦ ᖃᐅᔨᔪᖅ,エラーが見つかりました。, 오류가 발견되었습니다., tika atrasta kļūda., rasta klaida., znaleziono błąd, erro encontrado., foi encontrado um erro., была обнаружена ошибка., chyba bola nájdená., error encontrado., ข้อผิดพลาดที่พบ., hata bulundu.,знайдено помилку., lỗi đã được tìm thấy.
+,correct,(Correct and resubmit) ,(Corriger et resoumettre) ,,,,,,,,,,,,,,,,,,,,,,,,,,,,(corregir y reenviar) ,,,,
```

Comment:
Maybe use a more descriptive name, like "err-correct" or something else.
And double check the English translation

Reply:
This setting are now added with the name ```err-correct``` in the master spreadsheet.